src/FieldWarning/Assets/Model/Game/VisibleBehavior.cs

```diff
@@ -26,7 +26,7 @@ public class VisibleBehavior //: IComponentData
     private float stealth_factor = 10f;
 
     private HashSet<VisibleBehavior> _spotters = new HashSet<VisibleBehavior>();
-    private bool _isVisible = true;
+    public bool isVisible = true;
```

Comment:
Code style: PascalCase for methods and public fields, _camelCase for private variables.